File: LeetCode/Easy/0171-excel-sheet-column-number/0171-excel-sheet-column-number.go
func titleToNumber(columnTitle string) int {
    ret := 0
    power := len(columnTitle)
    for i := 0; i < len(columnTitle); i++ {
        pow := 1
        for j := 1; j < power; j++ {
            pow *= 26 
        }
        ret += (int(columnTitle[i]) - 64) * pow
        power--
    }
    return ret
}

File: LeetCode/Easy/0171-excel-sheet-column-number/0171-excel-sheet-column-number.java
class Solution {
    public int titleToNumber(String columnTitle) {
        int ret = 0;
        int power = columnTitle.length() - 1;
        for (int i = 0; i < columnTitle.length(); ++i){
            ret += (columnTitle.charAt(i) - 64) * Math.pow(26, power);
            --power;
        }
        return ret;
    }
}